anaconda.spec.in

```diff
@@ -37,7 +37,7 @@ Source0: %{name}-%{version}.tar.bz2
 %define libxklavierver 5.4
 %define mehver 0.23-1
 %define nmver 1.0
-%define pykickstartver 3.16.2-1
+%define pykickstartver 3.16.3-2.1
```

Comment:
Please fix the pykickstart version.
First release `2.1` is probably not correct and second there will be a new pykickstart build so the version will probably change.

Reply:
The new version should be `3.16.4-1`